index.js

```diff
@@ -573,8 +574,23 @@ class Raspi extends EventEmitter {
     throw new Error('reportDigitalPin is not yet implemented');
   }
 
-  pingRead() {
-    throw new Error('pingRead is not yet implemented');
+  pingRead(config, handler) {
+    const pulseOut = config.pulseOut;
```

Comment:
@sesh-kebab @nebrius 

It's the second arg passed to `pulseIn`

```
pulseIn(pin, HIGH);
```


> Reads a pulse (either HIGH or LOW) on a pin. For example, if value is HIGH, pulseIn() waits for the pin to go HIGH, starts timing, then waits for the pin to go LOW and stops timing. Returns the length of the pulse in microseconds or 0 if no complete pulse was received within the timeout.

(https://www.arduino.cc/en/Reference/PulseIn)

Looks like it's irrelevant to `Sonar`, so you can safely ignore it. I'll make a proper table for those properties